test/api-digester/source-stability.swift.expected

```diff
@@ -17,27 +13,176 @@ Protocol MutableIndexable has been removed (deprecated)
 Protocol RandomAccessIndexable has been removed (deprecated)
 Protocol RangeReplaceableIndexable has been removed (deprecated)
 
-/* More DictionaryIndex / SetIndex */
+/* FIXME: Bogus */
 Var Dictionary.endIndex has declared type change from DictionaryIndex<Key, Value> to Dictionary<Key, Value>.Index
 Var Dictionary.startIndex has declared type change from DictionaryIndex<Key, Value> to Dictionary<Key, Value>.Index
 Var Set.endIndex has declared type change from SetIndex<Element> to Set<Element>.Index
 Var Set.startIndex has declared type change from SetIndex<Element> to Set<Element>.Index
+Constructor BidirectionalSlice.init(base:bounds:) has 2nd parameter type change from Range<Base.Index> to Range<BidirectionalSlice.Index>
```

Comment:
It is surely ok to break the output, but would you please document that they are false positives? As an alternative, we should have a universal clang importer (and deserializer) in the future that can load any versions of modules and compare them in a fuller context. But, as for now, as you said, it seems too hard to eliminate false positives like these.